### PR TITLE
initial commit atom extension

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by kthoms** [comment]
How to test?

**Comment by cdietrich** [comment]
see README.md

File: atom-extension/grammars/mydsl.cson [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'scopeName': 'source.mydsl'
+'name': 'MyDslFiles'
+'fileTypes': [
+  'mydsl'
+]
+injections:
+  'L:*':
+    patterns: [
+        {
+            match: '\\b(Hello|from)\\b'
```
Comment (szarnekow) [inline atom-extension/grammars/mydsl.cson]: Is this handwritten or generated?

Comment (cdietrich) [reply atom-extension/grammars/mydsl.cson]: we dont have any fragments for such things yet.

**Pull request merged**